src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+
+import { ReactComponent as ArrowDropdown } from 'assets/images/icons/arrow-dropdown.svg';
+import { Answer } from 'types/answer';
+
+export const dropdownDataTestIds = {
+  base: 'dropdown__base',
+};
+
+interface DropdownProps {
+  questionId: string;
+  items: Answer[];
+  onValueChanged?: (value: Answer) => void;
+}
+const Dropdown = ({ questionId, items, onValueChanged }: DropdownProps): JSX.Element => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selectedValue, setSelectedValue] = useState(items[0].text);
+
+  const toggleDropdown = () => {
+    setIsOpen((prev) => !prev);
+  };
+
+  const handleOnSelectValue = (value: Answer) => {
+    setSelectedValue(value.text);
+    onValueChanged?.(value);
```

Comment:
this `onValueChanged` is making no sense now, in what case we don't pass the `onValueChanged` prop into this component?

Reply:
@nvminhtue I think we always pass `onValueChanged` to this component 👍 Let me update to make `onValueChanged` as a mandatory prop

Reply:
@nvminhtue fixed in [4803f51](https://github.com/manh-t/react-survey/pull/78/commits/4803f511251c858a44008f8f4f3580956f568ca8)